ci/rpm7/build.sh

```diff
@@ -0,0 +1,370 @@
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# Author: Dmitrii Demin
+
+url_src='https://github.com/telefonicaid/fiware-orion'
+url_dst='https://nexus.lab.fiware.org/repository/el'
+releasever=7
+basearch='x86_64'
+
+function _usage()
+{
+  echo -n "Usage: build [options]
+  Options:
+    -h   --help          show help
+    -b   --branch        specify branch/tag to build, if not specified - the source from /opt/fiware-orion will be used
+    -s   --source        specify repository to clone, if not specified - default repo (https://github.com/telefonicaid/fiware-orion) will be used
+    -p   --path          specify path to use as home, if not specified - /opt/fiware-orion will be used
+    -M   --make          cmake/make, stage (unit/functional) should be specified
+    -I   --install       make install
+    -o   --optional      run style, payload, file_compliance checks
+    -u   --unit          run unit tests
+    -f   --functional    run functional tests
+    -n   --nightly       build rpm nightly
+    -r   --release       build rpm release
+    -t   --testing       build rpm testing
+    -U   --upload        upload rpm, REPO_USER and REPO_PASSWORD ENV variables should be provided
+    -F   --fix           execute fix for jenkins and travis (disable ipv6 test)
+    -E   --execute       run (rerun) test stand with 2 orions
+    -D   --db            start mongodb
+    -S   --show          show the list of necessary commands that should be executed before starting functional tests manually
+
+  Examples:
+    build -M unit -IDu -b master   clone from master, make (for unit testing), make install, run mongo, execute unit tests
+    build -M functional -IDFf      get source from mounted folder, make (for functional testing), make install, run mongo, execute FIX, functional test
+"
+  exit 0
+}
+
+function _fix_jenkins()
+{
+    echo "Builder: fix jenkins"
+    cp ${path}/test/functionalTest/testHarness.sh /tmp/builder/bu/testHarness.sh
+    sed $'/DISABLED=(/s/$/\\\n          \'test\/functionalTest\/cases\/0000_ipv6_support\/ipv4_ipv6_both.test\' \\\/1' ${path}/test/functionalTest/testHarness.sh > /tmp/ft.tmp && mv -f /tmp/ft.tmp ${path}/test/functionalTest/testHarness.sh
```

Comment:
Please, add a FIXME line above this line about the disabled .test. Maybe some day, some FIXME-hunter could help to solve it ;)

Reply:
Great idea xD

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/85f9adb0d02f1a9d1906bea2198af60546f9b612